halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/services/v1/Daemon.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2016 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.halyard.cli.services.v1;
+
+import retrofit.RestAdapter;
+import retrofit.RetrofitError;
+import retrofit.client.OkClient;
+
+import java.net.ConnectException;
+
+public class Daemon {
+  public static DaemonService getService() {
+    return getService(false);
+  }
+
+  public static DaemonService getService(boolean log) {
```

Comment:
You're right, I've been kind of cheating myself by calling the `GlobalOptions` global, but then always passing them around. I'll fix this up.